src/containers/Main/AppWrapper.tsx

```diff
@@ -4,10 +4,22 @@ import { hot } from 'react-hot-loader'
 import Login from '../User/Login'
 import Verify from '../User/Verification'
 import { UserProvider } from '../../contexts/UserState'
+import { useLanguageProvider } from '../../contexts/Localisation'
 import NonRegisteredLogin from '../User/NonRegisteredLogin'
 import AuthenticatedContent from './AuthenticatedWrapper'
+import { Loading } from '../../components'
 
 const AppWrapper: React.FC = () => {
+  const { error, loading, selectedLanguage } = useLanguageProvider()
+
+  console.log('selectedLanguage', selectedLanguage)
+  if (error) {
+    console.error(error)
+    return <p>Can't load language provider. {error.message}</p>
```

Comment:
Yes, exactly! Can't use a missing strings file to notify about missing strings 😆

(I mean, it could be in the local defaults, but there's no point, since it would only ever show in English)